libraries/tools/kotlin-stdlib-gen/src/templates/SpecialJVM.kt

```diff
@@ -373,7 +373,22 @@ object CommonArrays : TemplateGroupBase() {
         include(ArraysOfPrimitives, PrimitiveType.defaultPrimitives)
         include(InvariantArraysOfObjects)
     } builder {
-        doc { "Returns new array which is a copy of the original array, resized to the given [newSize]." }
+        doc {
+            """
+            Returns new array which is a copy of the original array, resized to the given [newSize], truncating or padding
+            with nulls if necessary. For all indices that are valid in both the original array and the copy, the two arrays
+            contents will be identical. For all indices that are valid in the copy, but not in the original, the copy will
+            contain null.
+
+            Examples:
```

Comment:
It's good that you've provided some samples, but please use `@sample` tag. Check this [short guide](https://github.com/JetBrains/kotlin/tree/master/libraries/stdlib/samples) about sample authoring.